lib/tasks/test_replication.rake

```diff
@@ -55,16 +58,27 @@ class EvmTestSetupReplication
 
   private
 
+  def fetch_branch
+    if system("git fetch --depth=1 https://github.com/ManageIQ/manageiq.git refs/heads/darga:#{TEST_BRANCH}")
+      return true
+    end
+    system("git fetch --depth=1 git@github.com:ManageIQ/manageiq.git refs/heads/darga:#{TEST_BRANCH}")
+  end
+
   def released_migrations
-    json = Net::HTTP.get(URI('https://api.github.com/repos/ManageIQ/manageiq/contents/db/migrate?ref=darga'))
+    return [] unless fetch_branch
+    files = `git ls-tree -r --name-only #{TEST_BRANCH} db/migrate/`
+    return [] unless $CHILD_STATUS.success?
```

Comment:
Oh wait it's for this...I don't think you need to require 'English' for this...it should already be available.